shared/actions/teams.tsx

```diff
@@ -481,13 +484,10 @@ const ignoreRequest = async (action: TeamsGen.IgnoreRequestPayload) => {
       {name: teamname, username},
       Constants.teamWaitingKey(teamname)
     )
-    return false
   } catch (_) {
-    // TODO handle error, but for now make sure loading is unset
-    // TODO get rid of this once core sends us a notification for this (CORE-7125)
-    // TODO just unset loading without calling this
-    return TeamsGen.createGetDetails({teamname}) // getDetails will unset loading
+    // TODO handle error
```

Comment:
??

Reply:
I removed the part of the comment that isn't relevant anymore